react-ui/src/ControlPanel.jsx

```diff
@@ -55,13 +60,28 @@ export class ControlPanel extends PureComponent {
     this.setState({ open: !this.state.open })
   }
 
-  setPictureFilter = ({ value }) => {
+  setPictureFilter = (value) => {
     this.props.setPictureFilter(value)
     this.props.captureUserPicture()
   }
+
+  setPictureFilterWeight = ({ target }) => {
+    this.props.setPictureFilterWeight(target.value)
+    this.props.captureUserPicture()
+  }
+
+  newPictureFilterFromImage = ({ target }) => {
```

Comment:
Nit: Destructing events is slightly smelly since it isn't clear what the type of incoming object is - I'd keep the argument as `event` and address it via `event.target`